.travis.yml

```diff
@@ -29,6 +31,8 @@ install:
         pip install -U tox ;;
       docs)
         pip install -U sphinx ;;
+      quality)
+        pip install -rrequirements/quality.txt
```

Comment:
I was hoping we could keep this all under the control of Tox. However, I just realise we hard code:

```
[testenv]
deps = -rrequirements/test.txt
```

Which is actually wrong because, for example, we have a `docs.in/txt` for the `docs` case.

I'm not sure how to do this though. So, this is more a note if you're interested in looking into it.